mmv1/third_party/terraform/services/compute/resource_compute_router_peer.go.erb

```diff
@@ -881,6 +942,18 @@ func flattenNestedComputeRouterBgpPeerManagementType(v interface{}, d *schema.Re
 	return v
 }
 
+func flattenNestedComputeRouterBgpPeerMd5AuthenticationKey(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	if v == nil {
+		return nil
+	}
+
+	transformed := make(map[string]interface{})
+	transformed["name"] = v
+	//key value is not returned as it is a sensitive field
+	transformed["key"] = nil
```

Comment:
If it is not returned, setting the value to nil will cause the diff you're seeing during testing. To fix this, try removing this.

If removing it does not work, try setting the value of `transformed["key"]` to the current value in state of `key`.

Reply:
Fixed this